### PR TITLE
[Docs] aws-ec2 -- clarify aws public cert is already preloaded

### PR DESCRIPTION
**Pull request by skippy**
clarify, and make more explicit, the language around the default AWS public certificate

### CODE REVIEW

**Comment by skippy** [comment]
per #1582

File: website/source/docs/auth/aws-ec2.html.md [hunk 1]
```diff
@@ -242,10 +242,11 @@ endpoints.
...
 ### Varying Public Certificates
 
-The AWS public certificate which contains the public key used to verify the
-PKCS#7 signature varies for groups of regions. The default public certificate
-provided with the backend is applicable for many regions. Instances whose PKCS#7
-signatures cannot be verified by the default public certificate, can register a
+The AWS public certificate, which contains the public key used to verify the 
+PKCS#7 signature, varies for different AWS regions.  The primary AWS public 
+certificate, which covers most AWS regions, is already included in Vault and 
+does not need to be added.  Instances whose PKCS#7 signatures cannot be
```
Comment (jefferai) [inline website/source/docs/auth/aws-ec2.html.md]: Super pedantic items: there are two spaces after the period in your new text, and the comma between "included in Vault, can register" should not be there.

Comment (skippy) [inline website/source/docs/auth/aws-ec2.html.md]: good catches; fixed!

**Comment by jefferai** [comment]
Looks good!

**Pull request merged**